docker-compose.override.yml.dist

```diff
@@ -11,6 +11,9 @@ version: "2"
 #    app:
 #        volumes:
 #            - "~/.composer:/root/.composer"
+# Add this 2 volumes if you have weird permissions issues (But you won't gonna be able to see your cache on your host)
+#            - /app/var/cache
```

Comment:
it works? it seems to be a named volume that doesn't exists

Reply:
It creates a volume from this folder when starting, the result is that the folder is not mapped to file system anymore.